model/config.go

```diff
@@ -113,6 +113,9 @@ const (
 
 	FILE_SETTINGS_DEFAULT_DIRECTORY = "./data/"
 
+	IMPORT_SETTINGS_DEFAULT_DIRECTORY      = "./import"
+	IMPORT_SETTINGS_DEFAULT_RETENTION_DAYS = 30
```

Comment:
This is totally arbitrary. Waiting for some PM input on this.
Not really needed by this PR.